guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -1,10 +1,10 @@
 [[Using_ISS]]
 == Synchronizing Content Between {ProjectServer}s
 
-{ProjectName} uses Inter-{Project} Synchronization (ISS) in Disconnected mode to synchronize content {ProjectServer}s, or between organizations on {ProjectServer}.
+{ProjectName} uses Inter-{Project} Synchronization (ISS) to synchronize content between 2 {ProjectServer}s including those that are airgapped.
```

Comment:
The section below defines what airgapped is. using the term at this point, before definition, may be confusing